gto-support-db/src/androidTest/java/org/ccci/gto/android/common/db/AbstractDaoTest.java

```diff
@@ -113,6 +112,70 @@ public void testHaving() {
         cursor.moveToFirst();
 
         assertThat(CursorUtils.getString(cursor, RootTable.COLUMN_TEST), is("2"));
+
+        cursor.close();
+    }
+
+    @Test
+    public void testCompoundKey() throws Exception {
```

Comment:
yes and no, it does show the stack trace and a message about which assert fails within the test. Of course those messages aren't specific at the moment.

I added in this test now because I'm trying to think through an easier way to manage many to many relationships and I think it's going to change internally how I handle PK where clause generation.

I'll see about splitting this up a bit to be more centric on individual Dao methods instead.